packages/openapi3/src/openapi.ts

```diff
@@ -1325,7 +1325,10 @@ function createOAPIEmitter(
           !paramModels.has(type) &&
           !shouldInline(program, type)
         ) {
-          getSchemaOrRef(type, Visibility.Read);
+          getSchemaOrRef(
+            type,
+            Visibility.Read | Visibility.Create | Visibility.Update | Visibility.Delete
```

Comment:
why not `Visibility.all`

Reply:
feels like it should be the same as the valud in the `visibility-usage`

Reply:
Ah yes, I was testing. You're right it should be Visibility.All